edn/src/edn.rustpeg

```diff
@@ -105,37 +104,41 @@ keyword -> Value
     }
 
 #[export]
-list -> Value = "(" __ v:(__ value)* __ ")" {
+list -> Value = "(" v:(value)* ")" {
     Value::List(LinkedList::from_iter(v))
 }
 
 #[export]
-vector -> Value = "[" __ v:(__ value)* __ "]" {
+vector -> Value = "[" v:(value)* "]" {
     Value::Vector(v)
 }
 
 #[export]
-set -> Value = "#{" __ v:(__ value)* __ "}" {
+set -> Value = "#{" v:(value)* "}" {
     Value::Set(BTreeSet::from_iter(v))
 }
 
-pair -> (Value, Value) = k:(value) " " v:(value) ", "? {
+pair -> (Value, Value) = k:(value) v:(value) {
     (k, v)
 }
 
 #[export]
-map -> Value = "{" __ v:(pair)* __ "}" {
+map -> Value = "{" v:(pair)* "}" {
     Value::Map(BTreeMap::from_iter(v))
 }
 
 // It's important that float comes before integer or the parser assumes that
 // floats are integers and fails to parse
 #[export]
 value -> Value
-    = nil / boolean / float / bigint / integer / text /
+    = __ v:(nil / boolean / float / bigint / integer / text /
       keyword / symbol /
-      list / vector / map / set
+      list / vector / map / set) __ {
+    v
+}
+
+whitespace = (" " / "\r" / "\n" / "\t" / ",")
```

Comment:
No — `{:foo 5 :bar 3}` is totally normal. One would only add commas if they helped clarity, as in `{5 3, 3 2, 2 3}`.

(Swift does the same thing for numbers, and Rust might too — `1_234_567` is acceptable numeric syntax, as is `12_34_56_78` if you live on bizarro planet.)